webapp/tests/karma/unit/services/target-aggregates.js

```diff
@@ -0,0 +1,1101 @@
+describe('TargetAggregates service', () => {
+
+  'use strict';
+
+  chai.config.truncateThreshold = 0;
```

Comment:
Was this left in by mistake? I feel like it's a good safety valve, and I assume this will also disable the threshold for all subsequent tests?

Reply:
I think this is a necessary addition. It makes the the test failures reporting expand Object and Array values, instead of displaying `AssertionError: expected { Object (a, b, ...) } to deeply equal { Object (a, b, ...) `, much like [this code](https://github.com/medic/cht-core/blob/master/webapp/tests/karma/utils.js#L97) except directly from the "source" :)

Reply:
How about moving it to execute when the utils file is loaded? That way it applies to all tests and we can drop that custom code.

Reply:
I moved it to `utils` and removed the custom code.